db/schema.rb

```diff
@@ -1092,6 +1092,9 @@
     t.string "main_display_image_content_type"
     t.bigint "main_display_image_file_size"
     t.datetime "main_display_image_updated_at"
+    t.string "origin"
```

Comment:
I would remove this field for now, see review comment re: storing these as PracticePartners. I think we'll need to re-use Practice's functionality re: selecting a Facility or Office as origin. We can tackle that as part of the editor story though.